lib/dovico/version.rb

```diff
@@ -1,3 +1,3 @@
 module Dovico
-  VERSION = '1.0.0'
+  VERSION = '1.1.0'
```

Comment:
This can be done in a separated MR when you will release the new 1.1.0 version. (So we can easily `tag` the bump version commit)